Automattic/wp-calypso
# Transaction fee: Update copy on comparison grid

Related to #

## Proposed Changes

* This updates the copy for the transaction fee on the price comparison grid. Conversation in pau2Xa-598#comment-14306.

#### BEFORE
<img width="1257" alt="Screenshot 2023-10-13 at 6 13 23 PM" src="https://github.com/Automattic/wp-calypso/assets/1269602/59f90843-636c-4e80-83ae-c89a189e753f">


#### AFTER

**Desktop**

<img width="1279" alt="Screenshot 2023-10-13 at 7 17 11 PM" src="https://github.com/Automattic/wp-calypso/assets/1269602/663ba00e-df54-4411-b366-5a1b84d584b4">

**Mobile**

<img width="200" alt="Screenshot 2023-10-13 at 7 17 11 PM" src="https://github.com/Automattic/wp-calypso/assets/1269602/58109d63-f89b-433c-abd2-cedfc7464ef7">


## Testing Instructions



* Navigate to `/start/plans` and verify that in the plans comparison grid that the copy for the transaction fee matches what's shown in the screenshots above. Verify in both desktop and mobile.
* Verify translations exist in non-EN
<img width="1304" alt="Screenshot 2023-10-30 at 11 12 54 AM" src="https://github.com/Automattic/wp-calypso/assets/1269602/468d7711-fdd7-46b9-ac57-506feadf9966">
